Tasks/SonarQubePostTest/PRCA/ReportProcessor.ps1

```diff
@@ -40,17 +114,17 @@ function CreateProjectGuidAndPathMap
 function IsComponentFormatValid($tokens)
 {
     #expected format for component '[SonarQube project key]:[SonarQube project value]:[MSBuild project guid]:[file name relative to MSBuild project file path]'
-    $isFormatValid = $false
 
     if (!$tokens)
     {
         Write-Verbose "IsComponentFormatValid: tokens is invalid"
-        return $isFormatValid
+        return $false
```

Comment:
I suppose we can throw here instead of silently failing as well. Not sure why silent failure were preferred. I will change the code unless advised otherwise.